WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -16,6 +16,7 @@
 #import "FBPredicate.h"
 #import "FBXCodeCompatibility.h"
 #import "XCUIApplication+FBTouchAction.h"
+#import "XCUIElement+FBCaching.h"
```

Comment:
Unused import

Reply:
we need it to resolve `fb_isResolvedFromCache`

Reply:
same below

Reply:
Woops, you're right.